### PR TITLE
Operations page

### PR DESCRIPTION
**Pull request by swilliams**
Here's a start to help with on-boarding. Please take a look and make suggestions as you see fit. Especially from the newer folks: @jesdaigle @krantzinator @some15 @ruulito

### CODE REVIEW

File: docs/Team/Operations.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+We have our own space in Google Drive, [TW Dev](https://drive.google.com/drive/folders/0B4f4uAcTHp4ZfjU4b29mN3VsRFhEOXBBRG5wZXBLbV9mY2d6OGZfTUpRS09HWWMxMjc2Um8?usp=sharing). Google Docs and apps are a great way to collaborate on documentation and other writings that are not quite appropriate for publishing here on The Guidelines. Some examples include:
+
+* Guides or other information commonly shared with clients.
+* Documentation about processes that haven't been published to Guidelines, yet. (Once these have been published to Guidelines, the document should be archived in Drive.)
+* Internal documentation, such as client information that should not be shared.
+* Slide decks from presentations given at meetings, meetups, and conferences.
+* Running lists for meeting notes. 
+* Silly things like how each person pronounces 'GIF' and who prefers emacs over vim.
+
+You are empowered to add documents here, but it is up to all of us to keep it organized. This folder is shared with the whole dev team, but is findable by the company at large, so mind the permissions tab on the spreadsheet where we detail the weaknesses of upper management and how to exploit them.
```
Comment (krantzinator) [inline docs/Team/Operations.md]: i have negative connotations with the word "empowered," as it is an oft-abused buzzword. however, it actually is used very fittingly in this sentence, so if i'm the only one that has a negative gut reaction to the word i'm good with keeping it.

**Comment by krantzinator** [comment]
This looks good. 95% of my stress in the first few weeks came from project-specific stuff, so I have created issue #47 to address the work side of day-to-day.

File: docs/Team/Operations.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+#### Basecamp
+Basecamp is the communications and project management tool we use in our day to day work. There are two development centric Teams to participate in. [#Dev](https://3.basecamp.com/3592910/projects/2093999) is the main development group and there is a [Developer & Design Team](https://3.basecamp.com/3592910/projects/5561884) to collaborate with the Design team. 
+
+Campfires in Basecamp are for chatting, but due to the nature of chat, are not the place for conversations that need to have a more permanent history. Use a Message Board to discuss an important or specific topic. If it proves fruitful enough, these topics might be turned into Guidelines, blog posts, or something similar.
+
+<!-- ## Calendar
```
Comment (krantzinator) [inline docs/Team/Operations.md]: did you want some of us to jump in and finish out more sections here?

File: docs/Team/Operations.md (hunk 1 above)
Comment (swilliams) [reply docs/Team/Operations.md]: It is overused here... how about "encouraged"?

File: docs/Team/Operations.md (hunk 2 above)
Comment (swilliams) [reply docs/Team/Operations.md]: I put that there as a reminder... It should be a note/issue here.

File: docs/Team/Operations.md (hunk 1 above)
Comment (krantzinator) [reply docs/Team/Operations.md]: that was my first thought also

File: docs/Team/Operations.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+---
+
+[&larr; Home]({{ site.root }}/) &mdash; [Team]({{ site.root }}/Team)
+
+# The Day to Day
+
+## Resources
+As a company, we have a tendency to throw people in the deep end and see if they can swim. Our business is fast-paced so the traits that allow someone to thrive in that kind of situation are necessary, but we can at least take the alligators out of the deep-end.
+
+### Where Stuff Goes and Where to Find It Again
```
Comment (jesdaigle) [inline docs/Team/Operations.md]: This is, IMO, one of the biggest problems. We do have a ton of documentation but unless you know it exists, there's no way to find it. Unless told, for instance, I would have never known to go to recruiting --> root --> blah blah --> blah blah blah to find _client_folders_ in GDrive. If we bubble it more to the surface AND make giving permissions to new people part of an internal onboarding checklist, I think that would eliminate some of the "whoa, how come I've never seen this before?"

File: docs/Team/Operations.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+* Running lists for meeting notes. 
+* Silly things like how each person pronounces 'GIF' and who prefers emacs over vim.
+
+You are encouraged to add documents here, but it is up to all of us to keep it organized. This folder is shared with the whole dev team, but is findable by the company at large, so mind the permissions tab on the spreadsheet where we detail the weaknesses of upper management and how to exploit them.
+
+#### Basecamp
+Basecamp is the communications and project management tool we use in our day to day work. There are two development centric Teams to participate in. [#Dev](https://3.basecamp.com/3592910/projects/2093999) is the main development group and there is a [Developer & Design Team](https://3.basecamp.com/3592910/projects/5561884) to collaborate with the Design team. 
+
+Campfires in Basecamp are for chatting, but due to the nature of chat, are not the place for conversations that need to have a more permanent history. Use a Message Board to discuss an important or specific topic. If it proves fruitful enough, these topics might be turned into Guidelines, blog posts, or something similar.
+
```
Comment (vinnieking06) [inline docs/Team/Operations.md]: I'd add also to not feel obligated to respond to every ping right when you receive it, but to be courteous enough to return them within a day. Also if you need to crank, turn notifications off by putting basecamp into 'focus mode'.

File: docs/Team/Operations.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+* Documentation about processes that haven't been published to Guidelines, yet. (Once these have been published to Guidelines, the document should be archived in Drive.)
+* Internal documentation, such as client information that should not be shared.
+* Slide decks from presentations given at meetings, meetups, and conferences.
+* Running lists for meeting notes. 
+* Silly things like how each person pronounces 'GIF' and who prefers emacs over vim.
+
+You are encouraged to add documents here, but it is up to all of us to keep it organized. This folder is shared with the whole dev team, but is findable by the company at large, so mind the permissions tab on the spreadsheet where we detail the weaknesses of upper management and how to exploit them.
+
+#### Basecamp
+Basecamp is the communications and project management tool we use in our day to day work. There are two development centric Teams to participate in. [#Dev](https://3.basecamp.com/3592910/projects/2093999) is the main development group and there is a [Developer & Design Team](https://3.basecamp.com/3592910/projects/5561884) to collaborate with the Design team.
```
Comment (mmislam101) [inline docs/Team/Operations.md]: Again, should a public facing website have links to private things like basecamps?

Comment (swilliams) [reply docs/Team/Operations.md]: Yes. You can't access without credentials.

**Pull request merged**